kratos/python/add_mesh_to_python.cpp

```diff
@@ -232,7 +240,11 @@ pybind11::list GetValuesOnIntegrationPointsDouble( TObject& dummy,
     IntegrationPointsArrayType integration_points = dummy.GetGeometry().IntegrationPoints(
                 dummy.GetIntegrationMethod() );
     std::vector<double> values( integration_points.size() );
+    #ifdef KRATOS_USE_NEW_INTEGRATION_POINT_METHODS
```

Comment:
now this IS the big change.

only thing that deprecation doesn't really help here :-(